src/gui/properties/peerlistwidget.cpp

```diff
@@ -307,7 +308,7 @@ void PeerListWidget::loadPeers(BitTorrent::TorrentHandle *const torrent, bool fo
     }
 }
 
-QStandardItem* PeerListWidget::addPeer(const QString &ip, const BitTorrent::PeerInfo &peer)
+QStandardItem* PeerListWidget::addPeer(const QString& ip, BitTorrent::TorrentHandle *const torrent, const BitTorrent::PeerInfo &peer)
```

Comment:
Style fix:  I believe the ampersand is supposed to be to the right of the space to match similar code elsewhere. 
_____
(const QString **&ip,** BitTorrent::
_____